apps/examples/src/examples/speech-bubble/SpeechBubble/SpeechBubbleUtil.tsx

```diff
@@ -87,8 +89,7 @@ export class SpeechBubbleUtil extends ShapeUtil<SpeechBubbleShape> {
 	}
 
 	getGeometry(shape: SpeechBubbleShape): Geometry2d {
-		const newShape = getHandleinShapeSpace(shape)
```

Comment:
These two helpers were a big source of confusion for me. I get the need to switch the handles from normalized space to shape space and back, but it's easier to pull these out into a variable like "handleInShapeSpace" rather than having the concept of a shape with a handle that's sometimes normalized and sometimes not